website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -116,6 +117,72 @@ components: sources: host_metrics: {
 				unit:    "seconds"
 			}
 		}
+		cgroups: {
```

Comment:
We might want to specifically state what our cgroup support is. Which minimal Linux version do we require? Does this work for v1 or v2 cgroups? Stuff like that.